apiserver/facades/client/controller/controller_test.go

```diff
@@ -123,11 +123,12 @@ func (s *controllerSuite) TestAllModels(c *gc.C) {
 	// The results are sorted.
 	expected := []string{"controller", "no-access", "owned", "user"}
 	var obtained []string
-	for _, env := range response.UserModels {
-		obtained = append(obtained, env.Name)
-		stateEnv, err := s.State.GetModel(names.NewModelTag(env.UUID))
+	for _, userModel := range response.UserModels {
+		obtained = append(obtained, userModel.Name)
+		stateModel, release, err := s.StatePool.GetModel(userModel.UUID)
 		c.Assert(err, jc.ErrorIsNil)
-		s.checkEnvironmentMatches(c, env.Model, stateEnv)
+		defer release()
```

Comment:
I guess `checkEnvironmentMatches` doesn't call anything on the model which relies on the DB so calling `release` here is safe enough... probably not worth it though.